source/globalCommands.py

```diff
@@ -1188,6 +1188,36 @@ def script_review_endOfLine(self,gesture):
 	script_review_endOfLine.__doc__=_("Moves the review cursor to the last character of the line where it is situated in the current navigator object and speaks it")
 	script_review_endOfLine.category=SCRCAT_TEXTREVIEW
 
+	def script_review_currentSymbol(self,gesture):
+		info=api.getReviewPosition().copy()
+		info.expand(textInfos.UNIT_CHARACTER)
+		text = info.text
+		curLanguage = None
+		if config.conf['speech']['autoLanguageSwitching']:
+			for field in info.getTextWithFields({}):
+				if isinstance(field,textInfos.FieldCommand) and field.command=="formatChange":
+					curLanguage=field.field.get('language')
+		if curLanguage is None:
+			curLanguage = speech.getCurrentLanguage()
+		expandedSymbol = characterProcessing.processSpeechSymbol(curLanguage, text)
+		if expandedSymbol == text:
+			# Translators: Reported when there is no replacement for the symbol at the position of the review cursor.
+			ui.message(_("No symbol replacement"))
+			return
+		repeats=scriptHandler.getLastScriptRepeatCount()
+		if repeats == 0:
+			ui.message(expandedSymbol)
+		else:
+			# Translators: Character in the expanded symbol dialog.
+			ui.browseableMessage(_("Character: %s\n" % text) +

```

Comment:
The way these strings are constructed is unnecessarily complicated. 

Consider:
``` python
# Translators: Character and it's replacement used from the "Review current Symbol" command. Example:  "Character: ? Replacement: question"
message=_("Character: {}\nReplacement: {}").format(text, expandedSymbol)
languageDescription = languageHandler.getLanguageDescription(curLanguage)
# Translators: title for expanded symbol dialog. Example: "Expanded symbol (English)"
title=_("Expanded symbol ({})").format(languageDescription)
ui.browseableMessage(message, title)
```